### PR TITLE
Create Wallet with user generated passphrase

### PR DESCRIPTION
**Pull request by konradkonrad**
This is the minimal fix for #132 -- we still store the passphrase in a file. IMHO, this should not be done, but I had trouble handling the passphrase across requests: passing it as a cookie did not work, because the websocket connection does not share the same view on cookies with the http part of the webapp.

### CODE REVIEW

File: raiden_installer/web.py [hunk 1]
```diff
@@ -156,7 +163,33 @@ def _run_funding(self, configuration_file: RaidenConfigurationFile):
             self._send_status_update(f"Minting {transfer_token.ticker}")
             mint_tokens(w3, account, transfer_token)
 
+    def _run_create_wallet(self, **kw):
+        form = PasswordForm(passphrase1=kw.get("passphrase1"), passphrase2=kw.get("passphrase2"))
+        if form.validate():
+            self._send_status_update("Generating new wallet file for Raiden")
+            passphrase = form.data["passphrase1"].strip()
+            account = Account.create(passphrase=passphrase)
+            network_name = "mainnet"
```
Comment (fredo) [inline raiden_installer/web.py:172]: why not use the kw which is passed by the frontend for `network`

File: raiden_installer/web.py [hunk 2]
```diff
@@ -620,7 +665,8 @@ def post(self, configuration_file_name):
         [
             url(r"/", IndexHandler, name="index"),
             url(r"/configurations", ConfigurationListHandler, name="configuration-list"),
-            url(r"/setup/(mainnet|goerli)", SetupHandler, name="setup"),
+            url(r"/setup/(mainnet|goerli)/(.*)", SetupHandler, name="setup"),
+            url(r"/create_wallet", WalletCreationHandler, name="create_wallet"),
```
Comment (fredo) [inline raiden_installer/web.py:669]: where is the endpoint `create_wallet` called?

File: resources/templates/index.html [hunk 3]
```diff
@@ -27,7 +27,7 @@ <h1>
     Launch
   </button>
   {% else %}
-  <button class="link-button" data-link-url="{{ reverse_url('setup', 'mainnet') }}" id="setup-button" disabled>
+  <button class="link-button" data-link-url="{{ reverse_url('setup', 'mainnet', None) }}" id="setup-button" disabled>
```
Comment (fredo) [inline resources/templates/index.html:30]: One suggestion: We could call the create wallet method here. Then we could get rid of the if branch in the setupHandler

File: raiden_installer/web.py (hunk 1 above)
Comment (konradkonrad) [reply raiden_installer/web.py:172]: good catch, this shouldn't be here anymore!

File: raiden_installer/web.py (hunk 2 above)
Comment (konradkonrad) [reply raiden_installer/web.py:669]: [here](https://github.com/raiden-network/raiden-wizard/pull/148/files#diff-0ee0c1487407917328c8cac52295d9e7R93)

Comment (fredo) [reply raiden_installer/web.py:669]: But I mean, who is calling create wallet? Other than the redirect from the setupHandler I cannot see it. So Why not calling it directly from the frontend?

File: raiden_installer/web.py (hunk 1 above)
Comment (konradkonrad) [reply raiden_installer/web.py:172]: ah, it turns out, the `network_name` is missing and needs to be passed around

**Pull request merged**